src/pkg/gc/gc.go

```diff
@@ -66,8 +67,8 @@ func StartDockerClient(endpoint string) *docker.Client {
 }
 
 func DiskSpaceGC(intervalInSeconds uint64, policy GCPolicy) {
-	diskSpaceFetcher := &DiskSpaceFetcher{}
-	gocron.Every(intervalInSeconds).Seconds().Do(CleanAllWithDiskSpacePolicy, diskSpaceFetcher, policy)
+	diskSpaceFetcher = &DiskSpaceFetcher{}
```

Comment:
does `diskSpaceFetcher` need global scope? If so a comment explaining why could be good.